--- internal/api/rust/matrix_sdk_ffi/matrix_sdk_ffi.go.bak	2026-07-23 16:39:06.249000000 +0100
+++ internal/api/rust/matrix_sdk_ffi/matrix_sdk_ffi.go	2026-07-23 16:32:50.560000000 +0100
@@ -23496,20 +23496,21 @@
 // possible, instead of just storing IDs and coming back to the timeline
 // object to look up items.
 func (_self *Timeline) GetEventTimelineItemByEventId(eventId string) (EventTimelineItem, error) {
 	_pointer := _self.ffiObject.incrementPointer("*Timeline")
 	defer _self.ffiObject.decrementPointer()
 	res, err := uniffiRustCallAsync[*ClientError](
 		FfiConverterClientErrorINSTANCE,
 		// completeFn
 		func(handle C.uint64_t, status *C.RustCallStatus) RustBufferI {
 			res := C.ffi_matrix_sdk_ffi_rust_future_complete_rust_buffer(handle, status)
+			fmt.Fprintf(os.Stderr, "GetEventTimelineItemByEventId returned status %d, buffer of length %d\n", status.code, uint64(res.len))
 			return GoRustBuffer{
 				inner: res,
 			}
 		},
 		// liftFn
 		func(ffi RustBufferI) EventTimelineItem {
 			return FfiConverterEventTimelineItemINSTANCE.Lift(ffi)
 		},
 		C.uniffi_matrix_sdk_ffi_fn_method_timeline_get_event_timeline_item_by_event_id(
 			_pointer, FfiConverterStringINSTANCE.Lower(eventId)),
